### PR TITLE
funannotate_annotate RAM

### PR DESCRIPTION
**Pull request by TomHarrop**
This is failing on a large-ish genome on GA

### CODE REVIEW

File: tools.yml [hunk 1]
```diff
@@ -1190,7 +1190,7 @@ tools:
     mem: 40
   toolshed.g2.bx.psu.edu/repos/iuc/funannotate_clean/funannotate_clean/.*:
     cores: 8
-    mem: 40
+    mem: 64
```
Comment (bgruening) [inline tools.yml:1193]: @TomHarrop do you know how large? can we make it conditional based on the input size?

Comment (TomHarrop) [reply tools.yml]: this was the wrong line in the file, sorry :facepalm:

**Comment by TomHarrop** [comment]
@bgruening we've only had one of the last 200 jobs complete (and lots fail) where the input size was over 3250 MB. If this doesn't work I'll try using the number of lines in the gbk.

**Comment by cat-bro** [comment]
@TomHarrop input_size is in GB units

Would it be OK to try 62 as the large-input RAM instead of 64? 64 is going to be awkward on GA because most of the worker nodes do not have this.

**Comment by TomHarrop** [comment]
@cat-bro thanks, updated

**Review by cat-bro — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Pull request merged**